### DISCUSSION TITLE
Ionic React, o que acharam ?

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1573] [Discussão]
Agora o Ionic tem suporte oficial ao React usando o **react-dom** 🚀 

[https://ionicframework.com/blog/announcing-ionic-react/](https://ionicframework.com/blog/announcing-ionic-react/)

Minha opinião, não substitui o ReactNative mas pra PWAs é foda e facilita muito a vida❤️

**Comment by vhfmag** [comment]
Muito legal ver resultado do trabalho do time do Ionic sobre a plataforma web. Curti especialmente que eu não preciso adotar o Ionic (e o Ionic React) inteiro pra ter seus benefícios: em alguns minutos eu instalei o `capacitor` e o `pwa-elements` em um projeto já existente e tive acesso a uma UI pronta e de qualidade pra câmera com ótimas experiências de uso (é como se abrisse o app de foto do celular) e de desenvolvimento (pra tirar foto é só chamar uma função que retorna uma promessa com a foto 🤯).

Pena que o `capacitor` não tem suporte (que eu tenha visto) a gravar vídeo (o que é essencial pro app que eu tô fazendo no momento), mas pode ser uma oportunidade pro hacktoberfest 😊

Enfim, vejo como uma alternativa excelente ao React Native, desde que se esteja ciente dos trade-offs. Como tudo que envolve apostar na web, pode parecer apostar no azarão, mas a verdade é que plataformas proprietárias vem e vão, enquanto apostar na web é estar no ombro de gigantes e sempre fazer uso da infraestrutura e P&D de 30 anos, que só vai crescer. Mas né, eu sou fanboy hehe.
